.github/workflows/test.yml

```diff
@@ -156,10 +156,8 @@ jobs:
           name: cypress-coverage
           path: cypress-coverage
 
-      - name: Combine coverage
-        run: yarn coverage:ci
+      # - name: Combine coverage
+      #   run: yarn coverage:ci
```

Comment:
Remove these if not needed. Also, I think this was the only reason we were installing dependencies in this job.

Reply:
I have not cleaned the package.json scripts but have removed the entire coverage job and made the codecov step the part of other two jobs. Seems to work.